server/index.js

```diff
@@ -43,6 +44,10 @@ app.get('/runCode', (req, res) => {
 io.on('connection', (socket) => {
   socket.on('chat message', (message) => {
     // console.log('message: ', message);
+    if (message.userName === 'Guest') {
+      const idSlice = socket.id.slice(0, 5);
+      message.userName = `Guest(${idSlice}) :`;
+    }
```

Comment:
Wondering if we still need this since we have random user names being generated now.